pkg/kubelet/cri/remote/remote_image.go

```diff
@@ -49,7 +49,7 @@ func NewRemoteImageService(endpoint string, connectionTimeout time.Duration) (in
 
 	conn, err := grpc.DialContext(ctx, addr, grpc.WithInsecure(), grpc.WithContextDialer(dialer), grpc.WithDefaultCallOptions(grpc.MaxCallRecvMsgSize(maxMsgSize)))
 	if err != nil {
-		klog.Errorf("Connect remote image service %s failed: %v", addr, err)
+		klog.ErrorS(err, "Connect remote image service failed", "addr", addr)
```

Comment:
Let's use full names for log keys apart from some exceptions (like "err")
```suggestion
		klog.ErrorS(err, "Connect remote image service failed", "address", addr)
```